packages/core/src/external/surescripts/fhir/patient.ts

```
@@ -0,0 +1,63 @@
+import { BadRequestError } from "@metriport/shared";
+import { Patient, Reference } from "@medplum/fhirtypes";
+import { ResponseDetail } from "../schema/response";
+import { convertDateToString } from "@metriport/shared/common/date";
+
+export function getPatient(detail: ResponseDetail): Patient {
+  const name = getPatientName(detail);
+  const gender = getPatientGender(detail);
+  const birthDate = getPatientBirthDate(detail);
+
+  return {
+    resourceType: "Patient",
+    id: detail.patientId,
+    ...(name ? { name } : undefined),
+    ...(gender ? { gender } : undefined),
+    ...(birthDate ? { birthDate } : undefined),
+  };
+}
+
+export function getPatientReference(patient: Patient): Reference<Patient> {
```

maybe throw is patient id undefined or something